morpheus/_lib/src/llm/llm_node_runner.cpp

```diff
@@ -60,9 +60,9 @@ LLMNodeRunner::LLMNodeRunner(std::string name, input_mappings_t inputs, std::sha
     {
         const auto& external_name = inp.external_name;
 
-        if (specified_input_names_set.contains(inp.external_name))
+        if (specified_input_names_set.contains(inp.internal_name))
         {
-            throw std::runtime_error("Input '" + inp.external_name + "' is specified more than once");
+            throw std::runtime_error("Input '" + inp.internal_name + "' is specified more than once");
```

Comment:
Question: we're sure this is a bug with the existing code?

Reply:
Pretty sure. My `DuplicateInput` test wasn't throwing error before change. `specified_input_names_set` is not storing `internal_name` not `external_name` in line 104:
```
specified_input_names_set.insert(inp.internal_name);
```
Change doesn't break the existing unit tests.